sealer/cmd/inspect.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright © 2021 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/alibaba/sealer/image"
+	"github.com/alibaba/sealer/logger"
+	"github.com/spf13/cobra"
+)
+
+var clusterFilePrint bool
+
+// inspectCmd represents the inspect command
+var inspectCmd = &cobra.Command{
+	Use:   "inspect",
+	Short: "print the imageInformation or clusterFile",
+	Example: `sealer inspect kubernetes:v1.18.3
+	 	   sealer inspect -c kubernetes:v1.18.3`,
+	Args: cobra.ExactArgs(1),
```

Comment:
If you can execute the command `sealer inspect -c kubernetes:v1.18.3`, I do not think the args should be exact 1. Because -c is a flag and for the above case there is no args.